<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>site_creator</class>
 <widget class="QWidget" name="site_creator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>site_creator</string>
  </property>
  <widget class="QPushButton" name="createSite">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>create site</string>
   </property>
  </widget>
  <widget class="QLabel" name="title1">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>721</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;SITE &amp;amp; DIRECTORY INFO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>721</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;CSV EDITOR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>36</y>
     <width>721</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="dirInfo">
    <item>
     <layout class="QHBoxLayout" name="siteDir">
      <item>
       <widget class="QLabel" name="siteDir_label">
        <property name="text">
         <string>Site Root :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="siteDir_text">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="siteDir_button">
        <property name="text">
         <string>Browse</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="siteDir_lock">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="inDFile">
      <item>
       <widget class="QLabel" name="inDFile_label">
        <property name="text">
         <string>Input Data File:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inDFile_text">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="inDFile_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="inDfile_lock">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="name_type">
      <item>
       <layout class="QHBoxLayout" name="name">
        <item>
         <widget class="QLabel" name="name_label">
          <property name="text">
           <string>Station Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="name_text">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="tabelName">
        <item>
         <widget class="QLabel" name="tableName_label">
          <property name="text">
           <string>Table Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tableName_text">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="logType">
        <item>
         <widget class="QLabel" name="logType_label">
          <property name="text">
           <string>Logger Type : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="logType_text">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="arrayInfo">
      <item>
       <layout class="QHBoxLayout" name="arrays">
        <item>
         <widget class="QLabel" name="arrays_label">
          <property name="text">
           <string>Arrays:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="arrays_text">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="arrayId">
        <item>
         <widget class="QLabel" name="ArrayId_label">
          <property name="text">
           <string>Array Id:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ArrayId_box">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="arrayId_lock">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="badVal">
        <item>
         <widget class="QLabel" name="badVal_label">
          <property name="text">
           <string>Bad Value:  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="badVal_text">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="keyFile">
      <item>
       <widget class="QLabel" name="keyFile_label">
        <property name="text">
         <string>Array Key File: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="keyFile_text">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="keyFile_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="outDir">
      <item>
       <widget class="QLabel" name="outDir_label">
        <property name="text">
         <string>Output Directory :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outDir_text">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="outDir_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="qCDir">
      <item>
       <widget class="QLabel" name="qCDir_label">
        <property name="text">
         <string>QC Directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="qCDir_text">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="qCDir_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="errorDir">
      <item>
       <widget class="QLabel" name="errorDir_label">
        <property name="text">
         <string>Error Log Directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="errorDir_text">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="errorDir_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="therm_1">
      <item>
       <widget class="QLabel" name="therm_label_1">
        <property name="text">
         <string>Therm 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="therm_text_1">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="therm_button_1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="therm_2">
      <item>
       <widget class="QLabel" name="therm_label_2">
        <property name="text">
         <string>Therm 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="therm_text_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="therm_button_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="therm_3">
      <item>
       <widget class="QLabel" name="therm_label_3">
        <property name="text">
         <string>Therm 3:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="therm_text_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="therm_button_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>721</width>
     <height>163</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="csvstuff">
    <item>
     <layout class="QHBoxLayout" name="csvSel">
      <item>
       <widget class="QLabel" name="csvSel_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Csv Data Element :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="csvSel_box">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Select</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="csvTypePos">
      <item>
       <layout class="QHBoxLayout" name="dataType">
        <item>
         <widget class="QLabel" name="dataType_label">
          <property name="text">
           <string>Data Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="dataType_box">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="pos">
        <item>
         <widget class="QLabel" name="pos_label">
          <property name="text">
           <string>Input Array Position:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pos_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Row">
          <item>
           <widget class="QLabel" name="row_label">
            <property name="text">
             <string>name: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="row_text">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="coefs">
      <item>
       <widget class="QLabel" name="major_label_coef">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Coefficients :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="coef">
        <item>
         <widget class="QLabel" name="coef_label">
          <property name="text">
           <string>1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="coef_text">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="coef_2">
        <item>
         <widget class="QLabel" name="coef_label_2">
          <property name="text">
           <string>2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="coef_text_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="coef_3">
        <item>
         <widget class="QLabel" name="coef_label_3">
          <property name="text">
           <string>3:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="coef_text_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="coef_4">
        <item>
         <widget class="QLabel" name="coef_label_4">
          <property name="text">
           <string>4:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="coef_text_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="coef_5">
        <item>
         <widget class="QLabel" name="coef_label_5">
          <property name="text">
           <string>5:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="coef_text_5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="coef_6">
        <item>
         <widget class="QLabel" name="coef_label_6">
          <property name="text">
           <string>6:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="coef_text_6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="coef_7">
        <item>
         <widget class="QLabel" name="coef_label_7">
          <property name="text">
           <string>7:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="coef_text_7">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="qc">
      <item>
       <widget class="QLabel" name="major_label_qc">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Quality Control Pramaters:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="qcLow">
        <item>
         <widget class="QLabel" name="qcLow_label">
          <property name="text">
           <string>Low:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qcLow_text">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="qcHigh">
        <item>
         <widget class="QLabel" name="qcHigh_label">
          <property name="text">
           <string>High:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qcHigh_text">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="qcStep">
        <item>
         <widget class="QLabel" name="qcStep_label">
          <property name="text">
           <string>Step:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qcStep_Edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="header">
      <item>
       <widget class="QLabel" name="major_label_header">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Output Header:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hName">
        <item>
         <widget class="QLabel" name="hName_label">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="hName_text">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hUnits">
        <item>
         <widget class="QLabel" name="hUnits_label">
          <property name="text">
           <string>Units:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="hUnits_text">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hMeas">
        <item>
         <widget class="QLabel" name="hMeas_label">
          <property name="text">
           <string>Measurment Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="hMeas_text">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>title1</zorder>
  <zorder>label</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>createSite</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
